nixos/modules/services/networking/dnscrypt-proxy.nix

```diff
@@ -277,7 +277,8 @@ in
         ExecStart = "${dnscrypt-proxy}/bin/dnscrypt-proxy ${toString daemonArgs}";
         Restart = "always";
         User = "dnscrypt-proxy";
-
+        CapabilityBoundingSet = "CAP_NET_BIND_SERVICE";
```

Comment:
This seems redundant, given that we're already running as an unprivileged user. Adding to the ambient would suffice. However, the socket is setup by systemd and passed along to the daemon so it doesn't actually need it anyway. Please remove this or add a comment as to what it accomplishes.